modules/launcher/src/main/java/org/mule/module/launcher/coreextension/DefaultMuleCoreExtensionManager.java

```diff
@@ -167,14 +167,14 @@ public void setPluginClassLoaderManager(PluginClassLoaderManager pluginClassLoad
     }
 
     /**
-     * Creates a {@link AdaptedDeploymentListener}.
+     * Creates a {@link DeploymentListenerAdapter}.
      *
      * @param artifactDeploymentListener the artifactDeploymentListener to be adapted.
      * @param type: the artifact type.
-     * @return an AdaptedDeploymentListener.
+     * @return an DeploymentListener.
      */
-    AdaptedDeploymentListener getAdaptedArtifactDeploymentListener (ArtifactDeploymentListener artifactDeploymentListener, ArtifactType type)
+    DeploymentListener getDeploymentListenerAdapter (ArtifactDeploymentListener artifactDeploymentListener, ArtifactType type)
```

Comment:
Rename to createDeploymentListenerAdapter to indicate that calling it will return a new instance